### DISCUSSION TITLE
Deploy to firebase app distribution without a google play account

### DISCUSSION THREAD

**Discussion by kasvith** [codemagic-ci-cd/codemagic-docs#1524] [Q&A] [answered]
Code magic gives the following error on deployment when tries to deploy 

```
Firebase App Distribution: Error: This project is not linked to a Google Play account
```

I can manually upload to firebase app distribution btw

**Comment by Grummfy** [comment]
I have exactly the same question ;)

**Reply by kasvith** [reply]
:/ idk what to do even

**Comment by kasvith** [comment, marked as answer]
If you encounter this, change the build type to `apk`. 
`aab` require google play store to generate binaries

**Reply by kasvith** [reply]
@Grummfy here

**Reply by Grummfy** [reply]
thanks, I will try.

but codemagic shoulddocument it ;)

**Reply by Grummfy** [reply]
it solve my issue

**Reply by icarusdust** [reply]
Hi @Grummfy, it is already in the docs )) 

`In order to distribute an .aab to testers via Firebase App Distribution, your Firebase project must be linked to your Google Play account. More information is available [here](https://firebase.google.com/docs/app-distribution/android/distribute-console?apptype=aab#before_you_begin)`
https://docs.codemagic.io/yaml-publishing/firebase-app-distribution/#distribution-to-firebase

**Reply by Grummfy** [reply]
well, indeed. But it's not very obvious. Some warning could help in the interface or some bold text. Thanks
